packages/forma-36-react-components/src/components/Button/Button.css

```diff
@@ -20,57 +16,75 @@
   border-radius: var(--border-radius-medium);
   font-family: var(--font-stack-primary);
   font-size: calc(1rem * (14 / var(--font-base-default)));
+  font-weight: 500;
```

Comment:
do we need to adjust the normal font weight token for that?

Reply:
for what I understood from my conversation with @fabe 
we still don't know if we want to change the medium token to 500

This might get clearer when we work on Typography maybe, but Idk

What should I do?

Reply:
Since we're using system fonts probably `Segoe UI` and `Helvatica` are the most rendered fonts as one targets windows and the other targets mac osx.

both support font weights range from **100** to **900**

<img width="135" alt="Screenshot 2021-01-06 at 15 21 38" src="https://user-images.githubusercontent.com/6163988/103778898-3a80b600-5033-11eb-9963-b540f100a859.png">

This is fine for now, as we're not sure about the naming, and could be a breaking change. but I'll create a ticket for v4

Reply:
Thank you!